doc/articles/getting-started-counter.md

```diff
@@ -0,0 +1,322 @@
+        VerticalAlignment="Center">
+    <TextBlock AutomationProperties.AutomationId="HelloTextBlock"
+          Text="Hello Uno Platform"
+          HorizontalAlignment="Center" />
+  </StackPanel>
+</Page>
+```
+
+This defines a page with a background set to the theme resource `ApplicationPageBackgroundThemeBrush`, meaning it will adapt to the theme (`Dark` or `Light`) of the application. 
+
+The page contains a `StackPanel`, which will lay out controls in a vertical stack and is aligned in the center of the page, both horizontally and vertically. The `StackPanel` contains a single `TextBlock` control, which displays the text `Hello Uno Platform` and is aligned in the horizontal center of the `StackPanel`.
+
+## Add a Control
+
+We're going to replace the existing `TextBlock` with an `Image` but before we can do this, we need to add the image file to the application. Download [this SVG image]('Assets/icon.svg') and add it to the `Assets` folder inside the `Counter` project. At this point, you should rebuild and restart the application in order for the image to be included in the application package.
+
+> [!NOTE] 
+> If you're working in Visual Studio, select the newly added icon.svg file in the `Solution Explorer`, open the `Properties` tool window, and make sure the `Build Action` property is set to `UnoImage`. For other IDEs, no further action is required as the `unoapp` template automatically sets the `Build Action` to `UnoImage` for all files in the `Assets` folder.
+
+Including SVG files with the `UnoImage` build action will use `Uno.Resizetizer` to convert the SVG file to a PNG file for each platform. The generated PNG files will be included in the application package and used at runtime. For more information on using `Uno.Resizetizer` in Uno Platform, see [Get Started with Uno.Resizetizer](xref:Uno.Resizetizer.GettingStarted). 
+
+Now that we have the image file, we can replace the `TextBlock` with an `Image`. 
+
+- Open the `MainPage.xaml` file in the `Counter` project
+- Replace the `TextBlock` with the following `Image` element
+
+    ```xml
+    <Image Width="150"
+            Height="150"
+            Source="Assets/logo.png" />
+    ```
+- The `Width` and `Height` have been set on the `Image` to ensure the image is displayed at the correct size. The `Source` property has been set to the path of the image file inside the `Counter` project.
+
+## Change Layout
+
+The layout of the application uses a `StackPanel` which allows multiple controls to be added as children and will layout them in a vertical stack. An alternative to the `StackPanel` that is often used to control layout within an Uno Platform application is the `Grid`. The `Grid` allows controls to be laid out in rows and columns, and is often used to create more complex layouts.
+
+A `StackPanel` is a good choice for this application as we want the controls to be laid out vertically, one above the other. Let's go ahead and add the remaining controls for the counter.
```

Comment:
Have you yet shown an image representing what the layout trying to create will look like? This would help the reader picture why doing layout this way.

Reply:
Yes, counter app image added